sconify.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# declare the app entrypoint
+ENTRYPOINT="python /app/app.py"
+# declare an image name
+IMG_NAME="private-data-email-app"
+
+IMG_FROM=${IMG_NAME}:temp-non-tee
+IMG_TO=${IMG_NAME}:tee-debug
+
+# build the regular non-TEE image
+docker build . -t ${IMG_FROM}
+
+# run the sconifier to build the TEE image based on the non-TEE image
+docker run -it \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            registry.scontain.com:5050/scone-production/iexec-sconify-image:5.3.9 \
+            sconify_iexec \
```

Comment:
This is missing the latest sconifier options `--cli` and `--crosscompiler`. See the latest version of `global-jenkins-library`.

Reply:
Why not use the sconify.sh script from the Jenkins library instead of handling a duplicate here ?
Is there something blocking its use ?

Reply:
If we want to be able to produce local build,I suggest we keep a sconify script as short and consistent as possible, which use the sconify.args file, something like that https://github.com/iExecBlockchainComputing/python-hello-world/blob/main/cloud-computing/sconify.sh.

Reply:
Thanks :)
I agree  with Zied, some options are missing

Reply:
Alright I see them thanks ..!